packages/client/src/redux/reducers/adminReducer.ts

```diff
@@ -52,7 +80,11 @@ const updateQuestion = (
         ...state.admin,
         organizations: state.admin.organizations.map((o) => o.id === orgId ? {
           ...o,
-          questions: o.questions.map(q => q.id === question.id ? question : q)
+          questions: o.questions.map(q => q.id === question.id ? question : q),
+          fields: o.fields.map(f => fieldsIds.includes(f.id) ? {
+            ...f,
+            questionIds: [...f.questionIds, question.id]
```

Comment:
Seems to just add ids but i dont think it removes them, plus i think it will add ids anyway no matter if it already exists

Reply:
i did not find a good way, to remove them